sdk/Sdk/Tasks/EnhanceExtensionsMetadata.cs

```diff
@@ -24,12 +25,20 @@ public class EnhanceExtensionsMetadata : Task
         [Required]
         public string? OutputPath { get; set; }
 
+        [Required]
+        public ITaskItem[]? AdditionalExtensions { get; set; }
```

Comment:
curious about the naming choice for this? what about these extensions is additional?

Reply:
Extensions in addition to what we will scan by default. The basis of this change is that before we would copy the startup DLL to the worker extensions build output, which would then cause it to be included in that projects build for extension discovery.

With this change we can no longer rely on copying that file over pre-build. So, instead we have a separate step to scan and append that startup action.